recipes/r-olinkanalyze/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set version = '3.0.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-olinkanalyze
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/OlinkAnalyze_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/OlinkAnalyze/OlinkAnalyze_{{ version }}.tar.gz
+  sha256: a9e871ba44fca43d99eaec086b182e0d9fbd3cdd3fee1c2c437d0a9c36daeecd
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=4.0.0
+    - r-broom
+    - r-car
+    - r-dplyr
+    - r-emmeans
+    - r-forcats
+    - r-generics
+    - r-ggplot2
+    - r-ggpubr
+    - r-ggrepel
+    - r-lme4
+    - r-lmertest
+    - r-magrittr
+    - r-readxl
+    - r-rlang
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+  run:
+    - r-base >=4.0.0
+    - r-broom
+    - r-car
+    - r-dplyr
+    - r-emmeans
+    - r-forcats
+    - r-generics
+    - r-ggplot2
+    - r-ggpubr
+    - r-ggrepel
+    - r-lme4
+    - r-lmertest
+    - r-magrittr
+    - r-readxl
+    - r-rlang
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+
+test:
+  commands:
+    - $R -e "library('OlinkAnalyze')"           # [not win]
+    - "\"%R%\" -e \"library('OlinkAnalyze')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=OlinkAnalyze
+  license: AGPL-3.0-only
```

```suggestion
  license: GPL-3.0-only
```
CRAN says GPL 3 and the package doesn't contain a license file that says otherwise. I see you're one of the authors, maybe you have other information?

---

Thanks for the review and rising this point. I am investigating the licensing situation.